docs/source/reference/config.rst

```diff
@@ -14,12 +14,12 @@ Available fields and semantics:
 
 .. code-block:: yaml
 
-  # Custom spot controller resources (optional).
+  # Custom managed job controller resources (optional).
   #
-  # These take effects only when a spot controller does not already exist.
+  # These take effects only when a managed job controller does not already exist.
   #
   # Ref: https://skypilot.readthedocs.io/en/latest/examples/spot-jobs.html#customizing-spot-controller-resources
```

Comment:
Note: with redirect, tag #customizing-spot-controller-resources is stripped. Is it possible to support? Very minor.

Reply:
Yes! Just made the redirection to keep the tags, and also updated the link here.